common/v2/services/Store/DataManager/index.ts

```diff
@@ -1,2 +1,2 @@
 export { IDataContext, DataProvider, DataContext } from './DataProvider';
-export { useDatabase } from './useDatabase';
+export { DatabaseService } from './DatabaseService';
```

Comment:
This is only used within the `DataProvider`.
Let's keep a clean api for the module and remove the export.